lookout/style/typos/__main__.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import sys
+
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.style.typos.nn_prediction import create_and_train_nn_prediction_from_file
+from lookout.style.typos.utils import CORRECT_TOKEN_COLUMN, TYPO_COLUMN
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+    subparsers = parser.add_subparsers(help="Commands", dest="command")
+
+    def add_parser(name, help_message):
+        return subparsers.add_parser(
+            name, help=help_message, formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+
+    # Train NN prediction
+    vis_parser = add_parser("nn", "Train nn model for correction embedding prediction.")
+    vis_parser.set_defaults(handler=create_and_train_nn_prediction_from_file)
+    vis_parser.add_argument("fasttext_file", help="Path to binary dump of fasttext model.")
+    vis_parser.add_argument("data_file",
+                            help="Path to csv dump of pandas.DataFrame containing"
+                                 "columns [%s, %s]" % (CORRECT_TOKEN_COLUMN, TYPO_COLUMN))
+    vis_parser.add_argument("dump", help="Path to file to dump trained nn model.")
+    vis_parser.add_argument("-bs", "--batch_size", help="Batch size for training.")
```

Comment:
Short otions should be only one letter. Here it's probably better not to have a short option (just remove the first string). You should also add `type=int` argument to the `add_argument` call so that there is type conversion and checking done for you. You can also add a default value with `default=256` for example.